.claude/skills/new-release/scripts/04-catalog.sh

```
@@ -664,12 +677,25 @@ echo "📍 Step 2.7: Updating README.md..."
 
 README_FILE="README.md"
 if [[ -f "$README_FILE" && -n "$PREV_CATALOG_TAG" ]]; then
+<<<<<<< HEAD
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)